langstream-api/src/main/java/ai/langstream/api/runtime/ExecutionPlan.java

```diff
@@ -145,7 +145,7 @@ public Topic getTopicByName(String name) {
 
     public TopicDefinition getTopicDefinitionByName(String name) {
         return topics.entrySet().stream()
-                .filter(e -> e.getKey().getName().equals(name))
+                .filter(e -> e.getValue().topicName().equals(name))
```

Comment:
We lookup by topic name instead of  TopicDefinition name, Is it OK?

Reply:
if all the tests are passing then I guess that it is okay